### ISSUE TITLE
Error/warning log in console in cost models page

### ISSUE THREAD

**Issue by boaz0** [project-koku/koku-ui#1010]
**Describe the bug**
In development mode there are error and warning messages regard missing key/aria-label prop in some element.

**Expected behavior**
Although these messages don't mean that the system will crash immediately they do alert about a problem that will cause a failure in the future.

**Screenshots**
![Screenshot from 2019-09-23 17-15-25](https://user-images.githubusercontent.com/2453279/65433564-f2318a80-de25-11e9-9e69-69888507766a.png)

![Screenshot from 2019-09-23 17-15-55](https://user-images.githubusercontent.com/2453279/65433664-1e4d0b80-de26-11e9-8791-136b8e4b28f9.png)

![Screenshot from 2019-09-23 17-16-10](https://user-images.githubusercontent.com/2453279/65433583-f9589880-de25-11e9-9da9-70f13af43969.png)
